e2e/tests/caseManagementOrder_test.js

```diff
@@ -94,19 +94,11 @@ Scenario('Local Authority sends draft to Judge who requests corrections', async
   cmoHelper.assertCanSeeDraftCMO(I, caseViewPage, details);
 });
 
-
-//Skipped due to new error validation for approving a CMO with a hearing date in the future. We need to come up with
-// a better solution to account for this. Options:
-// - Have dynamic config to disable validation when e2es are run so it will allow us to skip the rules about submitting.
-// - Invoke the endpoint (not sure if jenkins have got access) to set the data with hearing date in past?
-// This would either require new endpoint on FPL or invoke the ccd endpoints.
-xScenario('Local Authority sends draft to Judge who approves CMO', async (I, caseViewPage, draftCaseManagementOrderEventPage, actionCaseManagementOrderEventPage) => {
-  // LA sends to judge
+Scenario('Local Authority sends draft to Judge who approves CMO', async (I, caseViewPage, draftCaseManagementOrderEventPage, actionCaseManagementOrderEventPage) => {
   await caseViewPage.goToNewActions(config.applicationActions.draftCaseManagementOrder);
   await cmoHelper.skipToReview(I);
   draftCaseManagementOrderEventPage.markToBeSentToJudge();
   await I.completeEvent('Submit');
-  I.dontSee('Draft orders', '.tabs .tabs-list');
```

Comment:
Note: draft orders tab is still visible after sending CMO to judge, this might be a bug that wasn't noticed as this test was commented. Will raise a separate ticket if it shouldn't be visible.

Reply:
is it just an empty tab?

Reply:
No, it contains all CMO data. Turns out it should still be visible in draft orders tab after sending to judge, but once he issues it then it's no longer draft and should be displayed in orders tab. Long story short, this step should be at the end of the test.